proxy.go

```diff
@@ -755,6 +756,21 @@ func SetLoggingLevel(l string) error {
 	return nil
 }
 
+// SetLoggingParams configures the logger
+func SetLoggingParams(logLevel string) error {
+	err := SetLoggingLevel(logLevel)
+	if err != nil {
+		return err
+	}
+
+	// enable nanosecond timestamps
+	proxyLog.Logger.Formatter = &logrus.TextFormatter{
+		TimestampFormat: time.RFC3339Nano,
```

Comment:
just whilst I'm here, not sure if directly related - I noticed our logs were putting in timezone info like `BST+1` etc. - would it be possible for us to move all the time logging to UTC, and hence drop the extra noise?

Reply:
That's happening for the runtime so we can look at that separately...

This change modifies the proxy timestamps from:

```
2017-09-28T08:52:45+01:00
```

to:

```
2017-09-29T08:52:45.176464812+01:00
```

The TZ is in there, but encoded numerically as an offset.